src/Symfony/Component/Security/Core/Authentication/Token/Storage/UsageTrackingTokenStorage.php

```diff
@@ -72,7 +73,28 @@ public function disableUsageTracking(): void
     public static function getSubscribedServices(): array
     {
         return [
-            'session' => SessionInterface::class,
+            'request_stack' => RequestStack::class,
         ];
     }
+
+    private function getSession(): SessionInterface
+    {
+        // BC for symfony/security-bundle < 5.2
+        if ($this->sessionLocator->has('session')) {
```

Comment:
Should we rename the `sessionLocator` property? `requestStackLocator`?

Reply:
what about just `container`?

Reply:
renamed into `container`